Src/CSharpier/SyntaxPrinter/SyntaxNodePrinters/Argument.cs

```diff
@@ -14,7 +14,11 @@ public static Doc Print(ArgumentSyntax node)
                 docs.Add(NameColon.Print(node.NameColon));
             }
 
-            docs.Add(Token.PrintWithSuffix(node.RefKindKeyword, " "));
+            if (node.RefKindKeyword.RawKind != 0)
```

Comment:
Is there a constant to compare to?
Or maybe a comment explaining the significance of `0` in `RawKind` would be helpful.